src/Filter/modules/MultiSelectFilter/MultiSelectFilter.tsx

```diff
@@ -28,7 +28,7 @@ const MultiSelectFilter = (
     }: MultiSelectFilterProps,
     multiSelectFilterOptions: MultiSelectFilterOptions = {}
 ): FilterModule<string[]> => ({
-    getFilterCount: (value) => value.length,
+    getFilterCount: (value) => value?.length || 0,
```

Comment:
I'm pretty sure @michaelpuzon 's comment above about `??` should apply here as well.  (I'm assuming it's mostly a stylistic suggestion that we should default to `??` to avoid errors in general, rather than that using `??` over `||` avoids a specific bug in that case.)

Reply:
Thanks! Got this one now as well.